test/TestUtils.py

```diff
@@ -77,25 +79,37 @@ def test_normalize_complex_command(self):
                                         'etc': 'whatever'})
         task3 = dict(name="hello", ec2="region=us-east1 etc=whatever")
         task4 = dict(name="hello", action="ec2 region=us-east1 etc=whatever")
-        self.assertEqual(utils.normalize_task(task1, 'tasks.yml'), utils.normalize_task(task2, 'tasks.yml'))
-        self.assertEqual(utils.normalize_task(task2, 'tasks.yml'), utils.normalize_task(task3, 'tasks.yml'))
-        self.assertEqual(utils.normalize_task(task3, 'tasks.yml'), utils.normalize_task(task4, 'tasks.yml'))
+        self.assertEqual(
+            utils.normalize_task(task1, 'tasks.yml'),
+            utils.normalize_task(task2, 'tasks.yml'))
+        self.assertEqual(
+            utils.normalize_task(task2, 'tasks.yml'),
+            utils.normalize_task(task3, 'tasks.yml'))
+        self.assertEqual(
+            utils.normalize_task(task3, 'tasks.yml'),
+            utils.normalize_task(task4, 'tasks.yml'))
 
     def test_normalize_args(self):
         task1 = dict(git={'version': 'abc'}, args={'repo': 'blah', 'dest': 'xyz'})
         task2 = dict(git={'version': 'abc', 'repo': 'blah', 'dest': 'xyz'})
 
         task3 = dict(git='version=abc repo=blah dest=xyz')
         task4 = dict(git=None, args={'repo': 'blah', 'dest': 'xyz', 'version': 'abc'})
-        self.assertEqual(utils.normalize_task(task1, 'tasks.yml'), utils.normalize_task(task2, 'tasks.yml'))
-        self.assertEqual(utils.normalize_task(task1, 'tasks.yml'), utils.normalize_task(task3, 'tasks.yml'))
-        self.assertEqual(utils.normalize_task(task1, 'tasks.yml'), utils.normalize_task(task4, 'tasks.yml'))
+        self.assertEqual(
+            utils.normalize_task(task1, 'tasks.yml'),
+            utils.normalize_task(task2, 'tasks.yml'))
+        self.assertEqual(
+            utils.normalize_task(task1, 'tasks.yml'),
+            utils.normalize_task(task3, 'tasks.yml'))
+        self.assertEqual(
+            utils.normalize_task(task1, 'tasks.yml'),
+            utils.normalize_task(task4, 'tasks.yml'))
 
     def test_extract_from_list(self):
         block = dict(
-                block = [dict(tasks=[dict(name="hello",command="whoami")])],
-                test_none = None,
-                test_string = 'foo'
+                block=[dict(tasks=[dict(name="hello", command="whoami")])],
```

Comment:
Let's maybe turn it into a proper dict literal then?

Reply:
Sure